src/client/app.coffee

```diff
@@ -82,32 +88,15 @@ for hexX in [0..12]
         hexagons.add hexagon
         uuidToHex.set hexagon.uuid, [hexX, hexY]
         hexToUuid.set String([hexX, hexY]), hexagon.uuid
+        validHexes.add String([hexX, hexY])
 
 scene.add hexagons
 
-class GameState
```

Comment:
seemed like it was overlapping with gameView

we can factor it back out later (with view handling rendering/input), gameState being UI/client independent